test/cppsuite/test_harness/workload_generator.h

```diff
@@ -41,8 +42,9 @@ namespace test_harness {
  */
 class workload_generator : public component {
     public:
-    workload_generator(configuration *configuration, workload_tracking *tracking)
-        : component(configuration), _tracking(tracking)
+    workload_generator(configuration *configuration, timestamp_manager *timestamp_manager,
+      workload_tracking *tracking)
+        : component(configuration), _timestamp_manager(timestamp_manager), _tracking(tracking)
     {
     }
 
```

Comment:
Delete the copy constructor of workload tracking and the timestamp manager to avoid coverity issues.